apps/test/unit/schoolInfo/hooks/useSchoolInfoTest.jsx

```diff
@@ -314,28 +344,6 @@ describe('useSchoolInfo', () => {
           PLATFORMS.BOTH
         );
       });
-
-      it('should clear the school name when CLICK_TO_ADD or NO_SCHOOL_SETTING are selected', () => {
-        act(() => {
-          hook.current.setSchoolId('2');
-        });
-        expect(hook.current.schoolName).toBe('Other School');
-
-        act(() => {
-          hook.current.setSchoolId(NonSchoolOptions.CLICK_TO_ADD);
-        });
-        expect(hook.current.schoolName).toBe('');
-
-        act(() => {
-          hook.current.setSchoolId('1');
-        });
-        expect(hook.current.schoolName).toBe('Cool School');
-
-        act(() => {
-          hook.current.setSchoolId(NonSchoolOptions.NO_SCHOOL_SETTING);
-        });
-        expect(hook.current.schoolName).toBe('');
-      });
```

Comment:
This is being deleted because now the school name is always deleted upon a school_id change right (not just for CLICK_TO_ADD or NO_SCHOOL_SETTING)?

Reply:
this test was added the first time I tried to fix this bug 😅 and it scaffolded some complex state management that seemed to make it more buggy tbh. in this pr, the `schoolName` does not get set when the `schoolId` changes anymore, so that makes this test irrelevant. 

it actually simplifies some of the form persistence we were trying to preserve, where if you accidentally make a bad selection, the manually entered `schoolName` is still there when you correct it

Reply:
Ahh gotcha gotcha, thank you for explaining!